projects/almalinux-8/almalinux-8-mysql-8/docker-compose.yml

```diff
@@ -43,10 +43,17 @@ services:
       context: ..
     depends_on:
       - catalog
+    volumes:
+      - shared_volume:/irods_testing_environment_mount_dir
 
   irods-catalog-consumer:
     build:
       context: ..
     depends_on:
       - irods-catalog-provider
+    volumes:
+      - shared_volume:/irods_testing_environment_mount_dir
+
+volumes:
+    shared_volume:
```

Comment:
Can you provide a brief explanation on what this does?

It doesn't have to be part of the code change. Just a simple explanation on the PR as a comment is fine.

Reply:
Does `shared_volume` become available on the host?

Reply:
`shared_volume` will appear as a Docker volume which will reside somewhere on the host (`/var/run/docker/...`). The name of the volume will have the Compose project name prepended, and will be cleaned up when the project is brought down along with the containers and network(s).

Reply:
Ok. Is it important for people to be able to find the `shared_volume` on the host?

Reply:
Not for any reason I can think of right now. Do you foresee something there?

Reply:
> Ok. Is it important for people to be able to find the shared_volume on the host?

no, me either.

Reply:
> Not for any reason I can think of right now. Do you foresee something there?

Not in particular. It sounds like this volume is something the user doesn't need to know about. Is that true?

Reply:
If a developer is implementing a test which uses this or even just wants to try out detached mode for unixfilesystem, they will need to know the volume mount inside the container. The actual volume isn't very important, though.

Reply:
Ok. So the volume's location on the host isn't important here. This is only important when inside of the container.

Got it.

Reply:
Do you want a comment here to explain what the volume mount is for?

Reply:
Yes. This feels like something we'll definitely forget a year from now. A comment would help with that situation.

Reply:
@JustinKyleJames - Once the comment is added explaining the purpose of the volume, I think we can merge this.

Reply:
Oh, and don't remove the ubuntu-20.04-mysql-5.7 project